### DISCUSSION TITLE
posts/2022-02-06-sv-sam

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#17] [General]
# posts/2022-02-06-sv-sam

A blog

https://cmdcolin.github.io/posts/2022-02-06-sv-sam

**Comment by godknows777** [comment]
I'm a rookie to this and this blog surely helps me a lot. Lots of appreciation.

**Reply by cmdcolin** [reply]
thanks for the kind words. understanding SAM/BAM/CRAM is fundamental to bioinformatics, yet somewhat underappreciated and buried under so many layers of analysis tools!

**Reply by godknows777** [reply]
yeah, you are right. I've been reproducing results from an article, and was totally confused at first. Another issue that keeps bothering me is installing the right version of softwares on my Linux server, I failed many times, and I'm still trying : )
